apps/i18n/common/en_us.json

```diff
@@ -1169,7 +1169,7 @@
   "manageStudentsNotificationFailure": "Something went wrong.",
   "manageStudentsNotificationCannotAdd": "You could not add {numStudents, plural, one {1 student} other {# students}} to your section. Please try again or refresh the page.",
   "manageStudentsNotificationCannotVerb": "The {numStudents, plural, one {student} other {students}} couldn't be {verb, select, copy {copied} move {moved} other {added}} to this section",
-  "manageStudentsNotificationAtCapacity": "You can’t have more than 500 students in a section. Questions about this limit? [Contact support](https://code.org/contact)",
+  "manageStudentsNotificationAtCapacity": "You can’t have more than {sectionCapacity} {sectionCapacity, plural, one {student} other {students}} in a section. Section {sectionCode} already has {sectionStudentCount} students in it{sectionSpotsRemaining, select, 0 {} other {, so you only can add up to {sectionSpotsRemaining} more {sectionSpotsRemaining, plural, one {student} other {students}} to it}}. Questions about this limit? [Contact support]({supportLink})",
   "manageStudentsNotificationAddSuccess": "You added {numStudents, plural, one {1 student} other {# students}} to your section.",
```

Comment:
Could you use more simple message format in this sentence?

Complicated variables and formats like `{sectionCapacity, plural, one {student} other {students}}` could confuse translators. A simple mistake with `{}` brackets can break our i18n pipeline. A very recent example is https://github.com/code-dot-org/code-dot-org/pull/40819/commits/7122922ef9ee78deac2d5f2f9f6855d8b1ca16b3, in which I had to manually change Azerbaijan translations in both json and CrowdIn.

Reply:
Hey @hacodeorg  - Should I also look to simplify line 1171? happy to simply pass the "verb" variable in rather than have the select statement?

@dju90 - Hoping to confirm with you on the wording as well. Ha let me know that overly complicated strings like this leave a lot of room for translation error, so simpler is better. Rather than using a plural format in the string (which would swap "student" for "students" in the event here are more than 1), is it appropriate to use "student(s)" as a single catch-all?

Reply:
yeah, go for it! "student(s)" is fine by me

Reply:
Yes, it'd be great to simplify 1171 also. Thank you and apologize for the late response.